tests/rules/_work.test.ts

```diff
@@ -49,6 +67,9 @@ const invalid: ReadonlyArray<RuleTester.InvalidTestCase> = [
   // }
 ];
 
+const olle = processInvalidTestCase(invalid);
+console.log("olle", olle);
+
```

Comment:
Is this just test code that can now be removed?

Reply:
Ah, good catch :-). That was just for testing, removed it.